src/Core/EventListener/ReadListener.php

```diff
@@ -96,15 +112,14 @@ public function onKernelRequest(RequestEvent $event): void
         }
 
         // TODO: 3.0 this is the default
-        if ($this->provider && isset($attributes['operation_name'])) {
+        if ($this->provider && null !== $operation->getName()) {
             try {
-                $identifiers = $this->extractIdentifiers($request->attributes->all(), $attributes);
-                $data = $this->provider->provide($attributes['resource_class'], $identifiers, $context);
+                $data = $this->provider->provide($attributes['resource_class'], $operation->getIdentifiers(), $context);
```

Comment:
La signature a changée:

```suggestion
                $data = $this->provider->provide($attributes['resource_class'], $operation->getIdentifiers(), $operation->getName(), $context);
```

Aussi il faut déclarer `$context` au dessus comme j'ai fais dans le WriteListener `$context = ['operation' => $operation];`